### PR TITLE
Added a Merch Tab linked to a google form with placeholder video and …

### PR DESCRIPTION
**Pull request by AlaqmarG**
# Changes 🛠

## What does this PR do?
- Merch tab to navigation bar
- Merch page with Buy Now button no top of a background video
- Buy Now button smooth scrolls to merch form
- Merch form sends responses to a google form

### Before
Before

### After
Home
Video
Black
White

## Any new npm dependencies?
NO

# Testing 🧪
Changes from the latest PR are deployed to https://brockcsc-pr.web.app once the checks are complete. Can use this for testing.

### Any other info needed for testing?
NO

### CODE REVIEW

File: src/app/views/merch/merch.component.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<div>
+  <div id="promo">
+    <button (click)="buynow(grid)" clickable>Buy Today!</button>
+    <video autoplay muted loop>
+      <source src="app/../assets/merch/merch.mp4" />
+    </video>
```
Comment (gideonoludeyi) [inline src/app/views/merch/merch.component.html:6]:
```suggestion
    <video autoplay muted loop>
      <source src="app/../assets/merch/merch.mp4" />
    </video>
    <button (click)="buynow(grid)" clickable>Buy Today!</button>
```

The button was not responsive for me. Maybe swap the order of the `<video>` and `<button>` elements instead, so the button is in the front.

File: src/app/views/merch/merch.component.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            *ngIf="
+              !form.controls.size.valid &&
+              (form.controls.size.touched || submitted)
+            "
+            >Please pick a size</span
+          >
+        </csc-input-container>
+        Interac e-transfer <b>45 CAD</b> to <b>payments@brockcsc.ca</b>.<br />
+        Locate the refence number on the application, or in the notification
+        email.
```
Comment (gideonoludeyi) [inline src/app/views/merch/merch.component.html:98]:
```suggestion
        Interac e-transfer <b>45 CAD</b> to <b>payments@brockcsc.ca</b>.<br />
        Locate the reference number on your banking app, or in the notification
        email.
```

May want to disambiguate what application is being referred to.

File: src/environments/environment.prod.ts [hunk 3]
```diff
@@ -52,4 +52,14 @@ export const environment: Environment = {
       message: 'entry.971826736',
     },
   },
+  merchForm: {
+    url: 'https://docs.google.com/forms/d/e/1FAIpQLSf3Aet204vFeXXP5IkWm0oEK_rm-VTvHFl6mwEXQxwuounRSg/formResponse',
```
Comment (gideonoludeyi) [inline src/environments/environment.prod.ts:56]: Can you create a separate form for receiving live orders?
Just thinking it'd make it easier distinguish real submissions from test submissions.

Comment (AlaqmarG) [reply src/environments/environment.prod.ts:56]: Fixed in d6b77705f30f9c6a1e68cd00d3ff467951dcc149

**Review by gideonoludeyi — APPROVED** [review]
:+1: This looks incredible

**Pull request merged**